packages/obs-backgroundremoval-git/obs-backgroundremoval-git.pacscript

```diff
@@ -13,27 +12,30 @@ pkgver() {
 version="$(pkgver)"
 
 prepare() {
+  # temporarily use latest buildable commit
+  git fetch --depth=10
+  git checkout bedcaf973424a05d2cfff1dc1295f5b516f82cd2
```

Comment:
```suggestion
```

Reply:
If we remove this, we will not use the latest git commit, which is not possible to build so it will not work

Reply:
Oh yeah, brain fart.